Dockerfile

```diff
@@ -92,6 +92,12 @@ RUN mkdir -p ${HOME}/.config/matplotlib && \
     echo "backend      : Agg" > ${HOME}/.config/matplotlib/matplotlibrc && \
     python -c "import matplotlib.pyplot"
 
+# Instruct joblib to use disk for temporary files. Joblib defaults to
+# /shm when that directory is present. In the Docker container, /shm is
+# present but defaults to 64 MB.
+# https://github.com/joblib/joblib/blob/0.11/joblib/parallel.py#L328L342
+ENV JOBLIB_TEMP_FOLDER=/tmp
```

Comment:
My inclination is to not accept this PR. This seems like an option that should be set by users at runtime or somewhere else in downstream process, not in a general, base container for doing things.